FinancialTracker/SceneDelegate.swift

```diff
@@ -46,8 +46,11 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
                     window.rootViewController = navVC
                 }
             } else {
-                if let alert = UIAlertController.create(basedOn: authError) {
-                    self.window?.rootViewController?.present(alert, animated: true)
+                if let authError = authError {
+                    let alertTitle = authError.title
+                    let alertMessage = authError.message
+                    
+                    self.window?.rootViewController?.present(UIAlertController.create(title: alertTitle, message: alertMessage), animated: true)
                     return
```

Comment:
Should there be a return here?

Reply:
Yes, if there is an error we shouldn't show any screen apart from the loading one. (In the scene delegate)

Reply:
if you want to use `return`, use `guard`